### DISCUSSION TITLE
Donations

### DISCUSSION THREAD

**Discussion by ippocratis** [Radiokot/photoprism-android-client#21] [Q&A] [answered]
How can we buy you coffees and beers?
:-)

**Comment by Radiokot** [comment, marked as answer]
I'm touched 😊 
Check out [tip options on my homepage](https://radiokot.com.ua/tip)

**Reply by ippocratis** [reply]
Unfortunately I don't have a crypto wallet and donating through the PayPal link throws "This payment cannot be completed due to regulations." Error
maybe KO-fi or GitHub sponsors or share your PayPal email so I could try directly transfer to your account?
Idk where that error came from I'm in Greece

**Reply by Radiokot** [reply]
I tried to update the settings so all the incoming funds will be converted to USD, I'm not sure if this fixes the error, but please try again. My PayPal email is spirit111@ukr.net

**Reply by ippocratis** [reply]
Yeah the link still returns the same error

I used your email instead

Keepup

**Reply by Radiokot** [reply]
I've received your donation! Thank you very much 😌

**Reply by Radiokot** [reply]
@ippocratis Ippokratis, would you like to be publicly added as a sponsor to the project description?

**Reply by ippocratis** [reply]
> @ippocratis Ippokratis, would you like to be publicly added as a sponsor to the project description?

Sure no problem
